code/script.js

```diff
@@ -229,97 +237,142 @@ const setSecret = () => {
 
 // This function to start (and restart) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+
+  // Hide win or lose section is visible from previous game
+  winOrLoseSection.style.display = 'none';
+
+  // This is so the that the first visible value in the select dropdown is not 'brown hair' but 'select'
+  questions.value="";
+
+  // Reset counter and remove number of guesses from previous game
```

Comment:
Nice that you got this function to work! ..(Makes the game more stressful...;))

Reply:
As you wrote in the "read" it would be fun with a sound effects. A melody that starts to play when the time is running out (if you have had that as a part of the game) or a warning signal that you have guessed 5 times already.

Reply:
I wish I coukd have given you some code suggestion for that;)

Reply:
Yes, one day we will get there with the sound effects!